Formula/llvm.rb

```diff
@@ -108,33 +128,54 @@ class Llvm < Formula
   keg_only :provided_by_osx
 
   option :universal
-  option "with-clang", "Build the Clang compiler and support libraries"
-  option "with-clang-extra-tools", "Build extra tools for Clang"
-  option "with-compiler-rt", "Build Clang runtime support libraries for code sanitizers, builtins, and profiling"
-  option "with-libcxx", "Build the libc++ standard library"
-  option "with-lld", "Build LLD linker"
+  option "without-clang", "Do not build the Clang compiler and support libraries"
+  option "without-clang-extra-tools", "Do not build extra tools for Clang"
+  option "without-compiler-rt", "Do not build Clang runtime support libraries for code sanitizers, builtins, and profiling"
+  option "without-libcxx", "Do not build the libc++ standard library"
+  option "with-libcxxabi", "Build the libc++abi standard library"
+  option "with-libunwind", "Build the libunwind library"
+  option "without-lld", "Do not build LLD linker"
   option "with-lldb", "Build LLDB debugger"
+  option "with-openmp", "Build additional OpenMP Runtime Libraries"
```

Comment:
:+1: for make it complete default. I think we can do the same thing for `clang`.